SubmersedVR/VR/VRQuickSlots.cs

```diff
@@ -86,6 +89,8 @@ void ArangeIconsInCircle(float radius)
                 var pos = CirclePosition(i, nSlots, radius);
                 icons[i].rectTransform.anchoredPosition = new Vector3(pos.x, pos.y, 0);
                 backgrounds[i].rectTransform.anchoredPosition = new Vector3(pos.x, pos.y, 0);
+                icons[i].rectTransform.LookAt(Camera.current.transform.position);
+                backgrounds[i].rectTransform.LookAt(Camera.current.transform.position);
```

Comment:
It was actually intentional that the quick wheel is flat up/down since I found it more natural to use.

Another way to do it would change or enable this right: https://github.com/Okabintaro/SubmersedVR/blob/master/SubmersedVR/VR/VRQuickSlots.cs#L174-L176
Thats how its done for vehicle but I found it much better being constrained and only rotate in the yaw when diving or on foot. Will have to test a bit more though.